source/contracts/reporting/Market.sol

```diff
@@ -322,10 +313,19 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
         return true;
     }
 
+    function doFeePayout(bool _toOwner, uint256 _amount) private returns (bool) {
+        if (_toOwner) {
+            getOwner().transfer(_amount);
+        } else {
+            // We can't do a simple transfer here since the reporting window is a delegated contract and the fallback function will fail on a standard send/transfer
```

Comment:
In general we should never be using `.send` or `.transfer` because it doesn't supply enough gas to contracts.  If that isn't the case, we should fix it.  If that is the case, I'm not sure if this comment is necessary.

Reply:
Note: If you find the comment valuable then I am not opposed to leaving it.

Reply:
I'll remove the comment. We aren't `Sending` or `Transfering` to contracts anywhere else to my knowledge.